### PR TITLE
Add airlines to Auto Suggest Input component

### PR DESCRIPTION
**Pull request by aaschlote**

### CODE REVIEW

File: data/airlines/data.json [hunk 1]
```diff
@@ -0,0 +1,214 @@
+[
+  { "name": "Aegean Airlines", "iata": "A3" },
+  { "name": "Aer Lingus Group Plc.", "iata": "El" },
```
Comment (pgurusinga) [inline data/airlines/data.json:3]: IATA should be EL https://www.iata.org/en/publications/directories/code-search/?airline.search=Aer%20Lingus

File: data/airlines/data.json [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+  { "name": "European Air Charter", "iata": "H6" },
+  { "name": "Cambodia Angkor Air", "iata": "K6" },
+  { "name": "Cathay Pacific Airways Limited", "iata": "CX" },
+  { "name": "China Airlines", "iata": "CI" },
+  { "name": "China Eastern Airlines Ltd.", "iata": "MU" },
+  { "name": "CityJet Limited", "iata": "WX" },
+  { "name": "Comair Limited", "iata": "MN" },
+  { "name": "Condor Flugdienst GmbH", "iata": "DE" },
+  { "name": "Corendon Airlines (Turkei)", "iata": "XC" },
+  { "name": "Corendon Airlines Europe (Malta)", "iata": "CGN" },
```
Comment (pgurusinga) [inline data/airlines/data.json:52]: CGN is not available. This should be XR https://www.iata.org/en/publications/directories/code-search/?airline.search=Corendon%20Europe

File: README.md [hunk 3]
```diff
@@ -160,3 +160,9 @@ In development mode, run the `npm run start:storybook` command.
 ### Airport
 
 `npm run build:airports-data`
+
+### Airline
+
+`npm run build:airlines-data "AIRLINE_FILE_PATH"`
+
+For the AIRLINE_FILE_PATH, please ask someone from the ZOV team about it!
```
Comment (eliflores) [inline README.md:168]:
```suggestion
The `AIRLINE_FILE_PATH` corresponds to the file that contains airlines data that will be transformed into `json` via the  `build:airlines-data ` task. For more information, please contact the maintainers of this repository.
```

File: data/airlines/data.json [hunk 4]
```diff
@@ -0,0 +1,214 @@
...
+  { "name": "Montenegro Airlines", "iata": "4O" },
+  { "name": "Neos S.p.A.", "iata": "NO" },
+  { "name": "Nesma Airlines", "iata": "NE" },
+  { "name": "Nordic Regional Airlines", "iata": "N7" },
+  { "name": "Norwegian Air International", "iata": "D8" },
+  { "name": "Norwegian Air Shuttle", "iata": "DY" },
+  { "name": "Nouvelair Tunisie", "iata": "BJ" },
+  { "name": "Oman Air", "iata": "WY" },
+  { "name": "Onur Air", "iata": "8Q" },
+  { "name": "Orange 2 Fly", "iata": "Q4" },
```
Comment (pgurusinga) [inline data/airlines/data.json:100]: should be it https://de.wikipedia.org/wiki/Orange2fly ?

File: scripts/airlines.ts [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+  const content = fs.readFileSync(filePath, { encoding: "utf-8" });
+
+  const rows = content.split("\n");
+  const airlines: Airline[] = [];
+  rows.forEach((row) => {
+    const airline = processAirlineRow(row);
+    airlines.push(airline);
+  });
+
+  // We need this extra airline due the flow process when the airline is not in the list
```
Comment (eliflores) [inline scripts/airlines.ts:40]:
```suggestion
  // For the use case that an airline is not part of the given airlines data, a `Sonstiges` (`Other`) airline is added to the list.
```

File: data/airlines/data.json (hunk 1 above)
Comment (pgurusinga) [reply data/airlines/data.json:3]: I think it's already correct 
```suggestion
  { "name": "Aer Lingus Group Plc.", "iata": "EI" },
```

File: app/components/inputs/autoSuggestInput/__test__/getDataListOptions.test.ts [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  });
+
+  it("it should return data options for airlines and find the LH airline", () => {
+    const actual = getDataListOptions("airlines");
+
+    expect(actual.length).greaterThan(0);
+    expect(actual.find((airline) => airline.value === "LH")).toBeTruthy();
+  });
+
+  it("it should return no data options in case parameter is not defined", () => {
```
Comment (eliflores) [inline app/components/inputs/autoSuggestInput/__test__/getDataListOptions.test.ts:18]:
```suggestion
  it("it should return no data options when the given argument is undefined", () => {
```

**Review by eliflores — APPROVED** [review]
LGTM 
Added some suggestions ✨

**Pull request merged**